google/ggrc-core
# GGRC-5338 Make methods in CMS.Models.DisplayPrefs synchronous

# Issue description

Actually methods in DisplayPrefs models work asynchronously. At the same time the application uses localStorage which is synchronous that is why no need to have these methods asynchronous.

# Steps to test the changes

Please check the following functionality:
1. recently viewed objects
2. restoring tree-view headers
3. restoring tree-view statuses
4. restoring create/edit modal state
5. restoring LNG state

# Solution description

Create recently-viewed-utils and display-prefs-utils which works synchronously instead of models.

# Sanity checklist




# PR Review checklist
